### ISSUE TITLE
Pasting error line into input is breaking CTRL-right EOL

### ISSUE THREAD

**Issue by rlamorea** [rlamorea/weBASIC#6]
Repro:

 1. Enter a line with an error
 2. Hit ENTER
 3. Get error pasted into new input (error highlighted)

Expected:
 - Can CTRL-right to jump to end of line

Actual:
 - CTRL-right does nothing

Analysis -- not resetting cursorEnd after pasting in text.

**Comment by rlamorea** [comment]
This is now fixed.

**Comment by rlamorea** [comment]
Reopening -- in fixing this I had also changed the SOL/EOL shortcuts to SHIFT-ALT left/right arrows because CTRL-arrow wasn't getting to the handler. But SHIFT-<any>-arrow needs to be reserved for highlighting later, so this was a bad choice.

I finally found the issue in this Reddit thread:
https://www.reddit.com/r/tmux/comments/s2x8pe/ctrlarrow_key_bindings_not_working_on_mac_works/

Mac configures "Mission Control" to eat the CTRL-left/right arrow sequences. These need to be disabled in order for the keys to get through to the browser to be handled.
